tests/server/blueprints/variants/test_controllers.py

```diff
@@ -38,3 +37,36 @@ def test_cancel_sanger_mail_sent(mock_mail, real_variant_database, institute_obj
     assert mock_mail._send_was_called
     ## THEN the supplied mail objects send method should have received a message object
     assert mock_mail._message
+
+def test_variant_csv_export(real_variant_database, case_obj):
```

Comment:
👍 for testing!

Reply:
Thanks, but I knew that you would asked me to test it anyway :)

Reply:
Haha